src/app/ui/NewTaskForm.tsx

```diff
@@ -54,6 +55,8 @@ export const NewTaskForm = ({
   const [filteredAssignees, setFilteredAssignees] = useState(filteredAssigneeList)
   const [activeDebounceTimeoutId, setActiveDebounceTimeoutId] = useState<NodeJS.Timeout | null>(null)
   const [loading, setLoading] = useState(false)
+  const [titleError, setTitleError] = useState(false)
+  const [assigneeError, setAssigneeError] = useState(false)
```

Comment:
I think there are a lot of unnecessary prop drills happening with these states. Since we are maintaining a store for this, it may be a better idea to carry this states to the store for better modularity.